### DISCUSSION TITLE
How do I install without using docker?

### DISCUSSION THREAD

**Discussion by ralyodio** [mudler/LocalAI#317] [Q&A] [answered]
I'm on linux.

**Comment by velanwins** [comment]
Hi did you get answer?

**Comment by velanwins** [comment]
Are you able to share the steps, if you already know

**Comment by bcamacho** [comment]
```make build```
```./local-ai --models-path ./models/ --debug```

**Comment by Aisuko** [comment, marked as answer]
Thanks @bcamacho It is one way to do it.

Here are two solutions:
1. You can download the pre compiled binary the official document is https://localai.io/basics/getting_started/#from-binaries
2. You can build it without Docker in your local environment. https://localai.io/basics/build/index.html#build-locally
